src/wcs1_1/src/main/java/org/geoserver/wcs/kvp/GetCoverageRequestReader.java

```diff
@@ -45,6 +47,17 @@ public Object read(Object request, Map<String, Object> kvp, Map<String, Object>
         // build output element
         getCoverage.setOutput(parseOutputElement(kvp));
 
+        String store = KvpUtils.caseInsensitiveParam(rawKvp, "store", null);
+        if (store != null) {
```

Comment:
This null check can be omitted if you swap the String comparison placing the static first

Reply:
Some users out there might be relying on this lenient behavior. Wondering if the stricter approach should be enabled when the strict OGC compliance flag is raised?
Mostly thinking out loud, as I don't personally believe many people are using store=true to start with, and setting store to other values would be the same as not setting it at all.

Reply:
@mprins - I don't understand what you are saying...

@aaime - I can add a check for strictness (I'll need the bean).  However, very few people will be using `store=...` let alone to an invalid value.  Right now, setting `store=foo` is the same as `store=false` which seems a bit strange that someone would actually set it but expect it to be false...

Reply:
As said, "mostly thinking out loud", wanted to check if you or @jodygarnett had an opinion on the topic.

Reply:
I'm fine either way - but I think its ok the way it is.  I can change the ticket title to "WCS STORE parameter - but be missing (false), 'true', or 'false'" as that will make it more obvious to people looking at release notes.

Reply:
@David I think @mprins is providing a coding style feedback: 
```
if (!"TRUE".equalsIgnoreCase(store) && !"FALSE"..equalsIgnoreCase(store)) {
  ...
}
```
Personally I find the ``store != null`` check to be clear.